Makefile

```diff
@@ -1,9 +1,9 @@
 ROOT_DIR:=$(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
-
 SAMPLE_DIR:=$(ROOT_DIR)/sample
-
 ZBOX=zbox
 ZBOXCLI=zboxcli
+CC=go
+SKD_COMMIT=f25689739bad70955675c19b9ef31ee236ab011f
```

Comment:
why it is fixed?

Reply:
because the error is given by your go env and not by the dependencies, basical in your go env you have an old version of the gosdk.

This is the problem that solved go 1.8 with workspace, but we don't have it so, this is the trick to fix it

Reply:
1. check go.mod,
1. if gosdk is replaced with local path, please go to the local path, and get last commitid as gosdk version
1. if gosdk is not replaced, just use gosdk version in require section